lib/controllers/situations.js

```diff
@@ -46,11 +46,8 @@ exports.create = function(req, res, next) {
 
 exports.simulation = function(req, res) {
     req.situation.simulate(function(err, result) {
-        if (err) {
-            res.status(err.code || 500).send(err);
-        } else {
-            res.send(result);
-        }
+        if (err) return res.status(err.code || 500).send(err);
```

Comment:
> This is not very readable.
> If you want to use an `if` without brackets, I still recommend you write the block line on a newline.
> In the very specific context of an error check, on the first line, that does an early exit, I believe it's acceptable to stay on one line (even though the chained methods here might make it a bit long).
> But in that case, it is important to keep an empty line after it. Otherwise, it is hard to read at first sight if the long line is a long condition and the block below is not indented, or if the long line is a short condition and a short block.